databuilder/models/table_owner.py

```diff
@@ -19,14 +19,14 @@ def __init__(self,
                  db_name,  # type: str
                  schema_name,  # type: str
                  table_name,  # type: str
-                 owners,  # type: List
+                 owners,  # type: str
                  cluster='gold',  # type: str
                  ):
         # type: (...) -> None
         self.db = db_name.lower()
         self.schema = schema_name.lower()
         self.table = table_name.lower()
-        self.owners = owners
+        self.owners = [owner.strip() for owner in owners.lower().split(',')]
```

Comment:
I think it works in this case, but could break in other cases which assume owners is a list.  So it is not very backward compatible... Given this pr is to enable owner dummy data load, do you think you could change the owners in the dummy loader to be a list and keep this file unchanged?

Reply:
@feng-tao I tried to do it initially but it didn't work for me
@jornh Done